node/full.go

```diff
@@ -341,6 +341,7 @@ func (n *FullNode) OnStart() error {
 		go n.blockManager.BlockSubmissionLoop(n.ctx)
 		go n.headerPublishLoop(n.ctx)
 		go n.blockPublishLoop(n.ctx)
+		return nil
```

Comment:
This is neat. I would say just a little bit _too neat_ ;)
Consider adding a short comment to explicitly state that **single** aggregator doesn't need `RetrieveLoop`, `BlockStoreRetrieveLoop` and `SyncLoop`.

Reply:
Added in the spec :)